README.md

```diff
@@ -66,8 +63,8 @@ After cloning this repository, build with following command.
 mvn package 
 ```
 
-Running Mini Instance
----------------------
+Running via Maven
```

Comment:
This can be made a seperate Fluo issue but how about reducing the output of the line below.  Like only have it print often if notifications are being queued or print it every 30 sec or minute, to know tablets are still being scanned.
```
2016-01-14 10:31:03,628 [hash.ScanTask] DEBUG: Scanned 1 of 1 tablets, added 0 new notifications (total queued 0)
```